elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -170,10 +170,22 @@ def stop(arguments={})
           unless pids.empty?
             print "\nStopping Elasticsearch nodes... ".ansi(:faint)
             pids.each_with_index do |pid, i|
-              begin
-                print "stopped PID #{pid}. ".ansi(:green) if Process.kill 'INT', pid
-              rescue Exception => e
-                print "[#{e.class}] PID #{pid} not found. ".ansi(:red)
+              ['INT','KILL'].each do |signal|
+                begin
+                  print "stopped PID #{pid} with #{signal} signal. ".ansi(:green) if Process.kill signal, pid
```

Comment:
This won't work as `Process.kill` returns `Fixnum` which is always evaluated as `true` in Ruby.

http://ruby-doc.org/core-1.9.3/Process.html#method-c-kill